js/ui/text_box/ui.text_editor.mask.strategy.base.js

```diff
@@ -129,16 +130,47 @@ export default class BaseMaskStrategy {
         const caret = this.editorCaret();
 
         editor._maskKeyHandler(event, () => {
-            const pastingText = getClipboardText(event);
+            const pastedText = getClipboardText(event);
             const restText = editor._maskRulesChain.text().substring(caret.end);
-            const accepted = editor._handleChain({ text: pastingText, start: caret.start, length: pastingText.length });
+            const accepted = editor._handleChain({ text: pastedText, start: caret.start, length: pastedText.length });
             const newCaret = caret.start + accepted;
 
             editor._handleChain({ text: restText, start: newCaret, length: restText.length });
             editor._caret({ start: newCaret, end: newCaret });
         });
     }
 
+    _autoFillHandler(event) {
+        const { editor } = this;
+        const inputVal = this.editorInput().val();
+        this._inputHandlerTimer = setTimeout(() => {
+            this._keyPressHandled = true;
+
+            if(this._isAutoFill()) {
+                this._keyPressHandled = true;
+
+                editor._maskKeyHandler(event, () => {
+                    editor._handleChain({ text: inputVal, start: 0, length: inputVal.length });
+                });
+                editor._validateMask();
+            }
+        });
+    }
+
+    _isAutoFill() {
+        const $input = this.editor._input();
+        let result = false;
+
+        if(browser.msie && browser.version > 11) {
```

Comment:
Does Edge v79+ (based on a Chromium) behave the same way?

Reply:
now it detected as a chrome

Reply:
{"webkit":true,"chrome":true,"version":"81.0"}